app/views/reasonableExcuseJourneys/other/UploadEvidencePage.scala.html

```diff
@@ -113,8 +114,11 @@
             <script type="text/x-template" id="error-manager-summary-item-tpl">@errorManager.summaryItem(messages)</script>
             <script type="text/x-template" id="error-manager-message-tpl">@errorManager.message(messages)</script>
 
-            <p>
-                @button("common.continue")
-            </p>
+            <div class="govuk-button-group">
+                <p>
```

Comment:
I don't think the `<p>` tag is needed here. In the non-JS versions, there is no `<p>` tag wrapping. Maybe worth keeping it consistent?